tests/NLog.UnitTests/Layouts/JsonLayoutTests.cs

```diff
@@ -68,6 +68,30 @@ public void JsonLayoutRendering()
 
             Assert.Equal("{ \"date\": \"2010-01-01 12:34:56.0000\", \"level\": \"Info\", \"message\": \"hello, world\" }", jsonLayout.Render(logEventInfo));
         }
+        
+        [Fact]
+        public void JsonLayoutRenderingIndentJson()
+        {
+            var jsonLayout = new JsonLayout()
+            {
+                Attributes =
+                    {
+                        new JsonAttribute("date", "${longdate}"),
+                        new JsonAttribute("level", "${level}"),
+                        new JsonAttribute("message", "${message}"),
+                    },
+                IndentJson = true
+            };
+
+            var logEventInfo = new LogEventInfo
+            {
+                TimeStamp = new DateTime(2010, 01, 01, 12, 34, 56),
+                Level = LogLevel.Info,
+                Message = "hello, world"
+            };
+
+            Assert.Equal("{\r\n\t\"date\": \"2010-01-01 12:34:56.0000\",\r\n\t\"level\": \"Info\",\r\n\t\"message\": \"hello, world\"\r\n}", jsonLayout.Render(logEventInfo));
```

Comment:
To make the unit-test work on non-Windows, then replace `\r\n` with `Environment.Newline`.